tests/Bus/Commands/ComponentGroup/AddComponentGroupCommandTest.php

```diff
@@ -28,9 +28,11 @@ class AddComponentGroupCommandTest extends AbstractTestCase
 
     protected function getObjectAndParams()
     {
-        $params = ['name' => 'Test', 'order' => 0, 'collapsed' => 1];
+        $params = ['name' => 'Test', 'order' => 0, 'collapsed' => 1, 'visible' => 2];
```

Comment:
Initially, I thought there were three possible values for visible: `0` viewable by public/guests
`1` only visible to logged in users, `2` hidden to logged in users but visible to its creator (see the description of this PR).

This is a place I forgot to modify.